swift/Rakefile

```diff
@@ -212,8 +212,8 @@ def target_add_slice2swift_build_rule(project, target, prefix = nil)
     rule.file_type = "pattern.proxy"
 
     script = <<~SCRIPT
-        if [ -f "${ICE_HOME-unset}/cpp/bin/slice2swift" ]; then
-            SLICE2SWIFT="$ICE_HOME/cpp/bin/slice2swift"
+        if [ -f "${ICE_HOME}/bin/slice2swift" ]; then
```

Comment:
I think we should keep it as `${ICE_HOME-unset}` in case ICE_HOME is not set. We don't want to accidentally hit `/bin/slice2swift`

Reply:
what is `-unset` for, is this a special keyword?

Reply:
It can be anything, it's just to provide a default in case the variable is not set not yet when you expect it to be. For example, it's often used to prevent against something like `rm -rf ${TYPOO}/`.